scripts/convert_to_csv.py

```diff
@@ -0,0 +1,40 @@
+import pyreadr
+import glob, os
+
+
+errors = []
+
+for fn in glob.glob("../data/rda_files/*.rda"):
+    try:
+        data = pyreadr.read_r(fn, timezone="PST")
+    except Exception as e:
+        errors.append(os.path.basename(fn))
+        continue
+    if not data:
+        errors.append(os.path.basename(fn))
+        continue
+    for df in data.values():
+        if os.path.basename(fn) == "ncdc_normals.rda":
+            df.to_csv(
+                os.path.join(
+                    "..",
+                    "data",
+                    "csv_files",
+                    f"{os.path.splitext(os.path.basename(fn))[0]}.csv.zip",

```

Comment:
This code uses `os.path.basename(fn)` several (4-5) times. At that point, it's a good idea to define a variable further up, something like
```
base_name = os.path.basename(fn)
```